### ISSUE TITLE
depth2struct generating masks for training

### ISSUE THREAD

**Issue by aj96** [tensorflow/models#6665]
For depth2struct, how and where are we supposed to save the masks for training depth2struct? The readme just says that it's assumed that the masks have been generated but it doesn't provide any information on how it is expecting the masks to be generated and where to save them.

Can someone please elaborate?

For reference, I am trying to train on kitti dataset. 

Thank you.

**Comment by ferdyandannes** [comment]
to generate the masks, you can use mask R-CNN to generate it

**Comment by aj96** [comment]
> to generate the masks, you can use mask R-CNN to generate it

Thank you. I know that. My question is how to save them and then how to load them for training. It isn't clear in the readme.

**Comment by ferdyandannes** [comment]
> > to generate the masks, you can use mask R-CNN to generate it
> 
> Thank you. I know that. My question is how to save them and then how to load them for training. It isn't clear in the readme.

To save them for the training, the masks image must be 1 channel image and save the image into -fseg file name.
for example your rgb image is 0000001.png the masks image must be 0000001-fseg.png

**Comment by aj96** [comment]
Thank you. For some extra clarification, all the masks for one image, say for 0000000001.png, should be all saved using the same pixel value, or it doesn't matter, it can be any non-zero value? And then 0000000001-fseg.png should be saved in the same folder as 0000000001.png. Then we can just run the training script? Because right now, I am getting a NoneType error which I am 99% sure is because I do not have the instance masks saved.

**Comment by zhangzhensong** [comment]
Please refer to [this issue](https://github.com/tensorflow/models/issues/6173).

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
